basics/Main.java

```diff
@@ -0,0 +1,86 @@
+// oracle docs https://docs.oracle.com/javase/8/docs/api/java/time/LocalDateTime.html
+import java.time.LocalDateTime;
+
+public class Main {
+  public static void main(String[] args) {
+    System.out.println("Hello World!");
+
+    // call pluralize
+    int rabbitCount = 0;
+    System.out.println("I own " + rabbitCount + " " + pluralize("rabbit", rabbitCount) + ".");
+    int pupCount = 1;
+    System.out.println("I own " + pupCount + " " + pluralize("pup", pupCount) + ".");
+    int catCount = 2;
+    System.out.println("I own " + catCount + " " + pluralize("cat", catCount) + ".");
+
+    // call Flipping Coins
```

Comment:
this wasnt mentioned so no points off, but generally when you are "testing" you code, you want to try it multiple times. try calling this method a few times with different numbers to make sure it works every time